doc/arm_jogging/arm_jogging_tutorial.rst

```diff
@@ -76,6 +76,10 @@ Settings
 --------
 User-configurable settings of the jog node are well-documented in ``jog_arm/config/ur_simulated_config.yaml``.
 
+Changing Control Dimensions
+---------------------------
+The dimensions jogging is allowed in may be changed with the ``change_control_dimensions`` (``moveit_msgs/ChangeControlDimensions``) service. The service consists of the 6 booleans representing the manipulator's degrees of freedom (3 translation and 3 rotation), defined in the input velocity command frame. Setting one of these booleans to ``false`` will result in the user input in that direction being overwritten with a 0, thus disallowing motion in the direction. 
+
```

Comment:
is there a tool for using this service? like RQT-based GUI?

Reply:
I am currently using this service in a QT GUI that includes functionality for our specific robot/task. 

I could possibly separate it into its own QT GUI, or create an RQT GUI for it

Reply:
if there isn't an easy out of the box suggestion to add to the tutorial, no problem. i was thinking this was more like dynamic reconfigure, but its not.